game.exs

```diff
@@ -1,7 +1,40 @@
 defmodule GuessingGame do
 
-  def hi() do
-    IO.puts("hello world")
+  def puts_high_low(n, t) do
+    if (n < t) do
+      IO.puts("Too Low")
+    else
+      IO.puts("Too High")
+    end
   end
 
+  def win() do
+    IO.puts("That's the right number, you win!")
+  end
+
+  def get_guess() do
```

Comment:
Read up on Elixir's pipes, I bet you could use them here if you relax the validation constraint.

For example, getting a guess is really just piping IO.gets to String.trim to String.to_integer... just what Elixir's pipes are for, you're gonna like them :)